emmet/vasp/settings/materials_settings.json

```diff
@@ -334,21 +415,69 @@
     },
     "tasks_key": "orig_inputs"
   },
+  {
+    "materials_key": "inputs.r2scan_opt",
+    "quality_score": {
+      "R2SCAN Structure Optimization": 2
+    },
+    "tasks_key": "orig_inputs"
+  },
+  {
+    "materials_key": "inputs.r2scan_static",
+    "quality_score": {
+      "R2SCAN Static": 6
+    },
+    "tasks_key": "orig_inputs"
+  },
+  {
+    "materials_key": "inputs.pbesol_opt",
+    "quality_score": {
+      "PBEsol Structure Optimization": 2
+    },
+    "tasks_key": "orig_inputs"
+  },
+  {
+    "materials_key": "inputs.pbesol_static",
+    "quality_score": {
+      "PBEsol Static": 6
+    },
+    "tasks_key": "orig_inputs"
+  },
   {
     "materials_key": "structure",
     "quality_score": {
-      "GGA+U Static": 4,
-      "GGA+U Structure Optimization": 3,
-      "GGA Static": 2,
-      "GGA Structure Optimization": 1
+      "GGA+U Static": 10,
+      "GGA+U Structure Optimization": 9,
+      "GGA Static": 8,
+      "GGA Structure Optimization": 7,
+      "PBEsol Static": 6,
+      "PBEsol Structure Optimization": 5,
+      "R2SCAN Static": 4,
+      "R2SCAN Structure Optimization": 3,
+      "SCAN Static": 2,
+      "SCAN Structure Optimization": 1
```

Comment:
I added these as alternatives for populating `.structure` because `.structure` is required to validate a materials doc. In the case of Cl2, my R2SCANs were apparently not structure matching to the GGA calcs, resulting in separate materials docs for just the R2SCAN structures. Without the change I made here, the `.structure` field would not be populated in that case, and the material would be invalidated.

Reply:
In my latest test built, this change results in ~450 non-deprecated materials that do NOT have a `gga` or `gga_u` entry. Instead, they just have a scan, r2scan, pbesol, or some combination of those.